addons/crm/models/crm_lead.py

```diff
@@ -1231,13 +1231,16 @@ def get_empty_list_help(self, help):
             help_title = _('Create a new lead')
         else:
             help_title = _('Create an opportunity to start playing with your pipeline.')
-        alias_record = self.env['mail.alias'].search([
-            ('alias_name', '!=', False),
-            ('alias_name', '!=', ''),
-            ('alias_model_id.model', '=', 'crm.lead'),
-            ('alias_parent_model_id.model', '=', 'crm.team'),
-            ('alias_force_thread_id', '=', False)
-        ], limit=1)
+        alias_domain = [
```

Comment:
That's not the same: you could have aliases creating opportunities, which is not what we expect here (only leads). The easiest thing would probably to search teams with an alias, then order them based on some criterion (instead of doing several searches), and take the first one in ordered result.